### DISCUSSION TITLE
Photoprism does not search for all people/faces in image

### DISCUSSION THREAD

**Discussion by amgschnappi** [photoprism/photoprism#2204] [Q&A]
Photoprism is set on my laptop using docker.

There are many photos which contain either single or multiple people.
For such image files.
1. Single person in image. That image is not seen in the People. Manually tagging also doesnt work, since nothing is shown in People tab. Picture is very clear.
![Single face example](https://user-images.githubusercontent.com/102602534/160628374-4a497d3d-5b95-4539-9973-f87d87a90246.PNG)

2. Multiple people in image. All faces are quite clear. But only some are suggested to be labelled in the People tab. (For eg: there are at least 6 clear faces in this image. But only 2 are suggested to be labelled.)
![Multi](https://user-images.githubusercontent.com/102602534/160626664-0ebd6709-09fe-4081-8a41-6cfd1eba243a.png)

**Comment by graciousgrey** [comment]
Facial detection is never perfect. Even Google photos does not detect each and every face. 

See also: https://docs.photoprism.app/user-guide/organize/people/#when-a-face-was-not-detected

Manual tagging as well as improving face detection is planned see: https://github.com/photoprism/photoprism/issues?q=is%3Aissue+is%3Aopen+people+in%3Atitle :)
